### PR TITLE
(DOCSP-26166) Copy prereq from SwiftUI tutorial to React Native tutorial

### PR DESCRIPTION
**Pull request by krollins-mdb**
## Pull Request Info

This PR adds more prerequisite info to the React Native Tutorial. The Swift tutorial had excellent relevant content for this, so I just pulled it over. ~~Long term, we should look at standardizing these prerequisite sections - potentially refactoring common bits like what's added here into an include that can be shared.~~

I abstracted the generic prerequisites to an include and updated the other tutorials with it.

### Jira

- https://jira.mongodb.org/browse/DOCSP-26166

### Staged Changes (Requires MongoDB Corp SSO)

- [React Native Tutorial > Prerequisites](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-26166/tutorial/react-native/#prerequisites)
- [Flutter Tutorial > Prerequisites](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-26166/tutorial/flutter/#prerequisites)
- [SwiftUI Tutorial> Prerequisites](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-26166/tutorial/swiftui/#prerequisites)
- [Kotlin Tutorial > Prerequisites](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-26166/tutorial/kotlin/#prerequisites)
- [Xamarin Tutorial > Prerequisites](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-26166/tutorial/dotnet/#prerequisites)
- [Triggers, Functions, and Values Tutorial > Prerequisites](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-26166/tutorial/backend/) - just the section name

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by nlarew — APPROVED** [review]

**Comment by mongodben** [comment]
can you also add this section to other tutorials that don't have it? (either in this or separate PR)

**Comment by krollins-mdb** [comment]
@mongodben, I had thought about doing that in another PR. But I guess it makes sense to just do it here! Updating now.

File: source/tutorial/kotlin.txt [hunk 1]
```diff
@@ -39,6 +39,8 @@ around 30 minutes.
 
 .. include:: /includes/kotlin-install-prerequisites.rst
```
Comment (nlarew) [inline source/tutorial/kotlin.txt:40]: Not sure why this is a separate include file - this is the only spot we use it and it obscures the "Prerequisites" heading.

Mind getting rid of the include and inlining the content here?

**Review by nlarew — APPROVED** [review]
LGTM! Thanks for doing a wider pass on this.

I left one non-blocking additional comment if you don't mind

**Review by nlarew — APPROVED** [review]

**Pull request merged**